src/libraries/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -393,14 +394,23 @@ public bool Equals(Complex value)
 
         public override int GetHashCode() => HashCode.Combine(m_real, m_imaginary);
 
-        public override string ToString() => $"<{m_real}; {m_imaginary}>";
+        public override string ToString() => ToString(null, null);
 
         public string ToString([StringSyntax(StringSyntaxAttribute.NumericFormat)] string? format) => ToString(format, null);
 
         public string ToString(IFormatProvider? provider) => ToString(null, provider);
 
-        public string ToString([StringSyntax(StringSyntaxAttribute.NumericFormat)] string? format, IFormatProvider? provider) =>
-            $"<{m_real.ToString(format, provider)}; {m_imaginary.ToString(format, provider)}>";
+        public string ToString([StringSyntax(StringSyntaxAttribute.NumericFormat)] string? format, IFormatProvider? provider)
+        {
+            // $"<{m_real.ToString(format, provider)}; {m_imaginary.ToString(format, provider)}>";
+            var handler = new DefaultInterpolatedStringHandler(4, 2, provider, stackalloc char[512]);
+            handler.AppendLiteral("<");
```

Comment:
There's no need for a `AppendLiteral` overload that takes `char`* as the JIT will optimize the copy to handler's buffer to be just a simple memory write. Correct?

\* beside the question if such an overload is relevant 😉

Reply:
> Correct?

a) Correct
b) Even if the JIT didn't, the method could (and previously did) use IsKnownConstant to special-case a single-character literal itself.
C) These APIs are really intended to be targeted by Roslyn, which emits a string.